playbooks/sendmail.yml

```diff
@@ -0,0 +1,20 @@
+---
+# by default this playbook runs in the staging environment
+# to run in production, pass '-e runtime_env=production'
+- name: build the ponyexpress
+  hosts: libpony_{{ runtime_env | default('staging') }}
+  remote_user: pulsys
+  become: true
+  vars_files:
+    - ../group_vars/sendmail/vars.yml
+    # - ../group_vars/sendmail/{{ runtime_env | default('staging') }}.yml
+    - ../group_vars/sendmail/vault.yml
+  roles:
+    - role: roles/sendmail
```

Comment:
The PR doesn't add a `sendmail` role. Is this outdated?

Reply:
removed those. PR started out as sendmail and OIT said we didn't need to use sendmail.